assets/src/edit-story/components/form/text.js

```diff
@@ -89,13 +88,27 @@ function TextInput({
   flexBasis,
   disabled,
   clear,
```

Comment:
Not related to these changes, but I found the name 'clear' not easily parsable, perhaps this should be named 'isClearIconVisible'.

Reply:
I agree. But as this is used in other places as well, I'll keep it like that.